src/help/util.ts

```diff
@@ -41,10 +41,10 @@ function collateSpacedCmdIDFromArgs(argv: string[], config: IConfig): string[] {
   if (argv.length === 1) return argv
 
   const findId = (argv: string[]): string | undefined => {
-    const ids = collectUsableIds(config.commandIDs)
+    const ids = new Set(collectUsableIds(config.commandIDs))
```

Comment:
@peternhale would it be better to have `collectUsableIds` return a `Set`? That way other consumers of it get the performance benefits?

Reply:
I'll take a look, but yes. would also save the sort op in collectUsableIds